b2share/modules/deposit/mappings/deposits/deposits.json

```diff
@@ -101,13 +114,28 @@
             "type" : "string",
             "index": "not_analyzed"
           },
-          "resource_type" : {
-            "type" : "string"
+          "resource_types" : {
+            "properties" : {
+              "resource_type" : {
+                "index":"analyzed",
+                "type" : "string",
+                "analyzer":"english"
+              },
+              "resource_type_general" : {
+                "index":"analyzed",
+                "type" : "string",
+                "analyzer":"english"
+              }
+            }
           },
-          "title" : {
-			"index":"analyzed",
-            "type" : "string",
-			"analyzer":"english"
+          "titles" : {
+            "properties" : {
+              "title" : {
+                "index":"analyzed",
+                "type" : "string",
+                "analyzer":"english"
+              }
+            }
```

Comment:
Missing
- `contributors.contributor_type`which should be `not_analyzed`.
- `language` which should be `not_analyzed`.
-  `description.description_type` which should be `not_analyzed`.